packages/web-components/src/presence-badge/presence-badge.styles.ts

```diff
@@ -0,0 +1,96 @@
+import { css } from '@microsoft/fast-element';
+
+export const styles = css`
+  :host {
+    align-items: center;
+    box-sizing: border-box;
+    cursor: default;
+    display: inline-flex;
+    flex-direction: row;
+    font-size: 0;
+    line-height: 0;
+    min-height: 6px;
+    min-width: 6px;
+    position: relative;
+    transition: opacity 0.5s linear;
+    --presence-size: 16px;
+    --presence-mask-width: 2px;
+    --presence-mask-color: #eee;
+    --presence-color-available: #92c353;
+    --presence-color-away: #eaa300;
+    --presence-color-busy: #d13438;
+    --presence-color-dnd: #d13438;
+    --presence-color-offline: #929292;
+    --presence-color-outofoffice: #c239b3;
+    --presence-color-unknown: #adc0cf;
+    --presence-color-blocked: #d13438;
```

Comment:
@bheston @nicholasrice do you think any of the above should be "tokens"?

Reply:
I suspect the `size` should be based on typography and density once that's worked out. `mask-width` could possibly be `stroke-width` or derived from that or `design-unit`. `mask-color` should use the container `fill-color` for now, but that's not going to work long-term or over non-solid surfaces. The colors I think should stay in this component. I don't think they are common enough to surface up.

Reply:
I revised my opinion on this in [fast pr 4882](https://github.com/microsoft/fast/pull/4882). I think these colors should adjust for light or dark mode. We should be able to do this by creating tokens and associating them with copies of the color rules, in this case similar to accent foreground since they are non-neutral colors that should retain contrast with their background. I'm not sure where those would be registered off the top of my head though. I don't think they belong in `design-tokens` but maybe @nicholasrice can clarify how those could be added only when a badge is composed.